### DISCUSSION TITLE
Using Copilot with vscode, how can I get copilot to recognize the languages in the code fences.

### DISCUSSION THREAD

**Discussion by alexhallam** [quarto-dev/quarto-cli#8786] [Q&A] [answered]
### Description

I just want copliot autocompletion while using Quarto. What I have to do instead I use separate files for `*.py`, `*.r` and `*.md`. Is there a what to get copilot smarter on Quarto.

**Comment by mcanouil** [comment, marked as answer]
There is nothing to do, it should work unless you deactivated Copilot for Quarto documents.
Note that it is unrelated to Quarto itself or even the Quarto extension for VSCode.


https://github.com/quarto-dev/quarto-cli/assets/8896044/19dd341f-623f-4162-b587-632048421499

**Comment by alexhallam** [comment]
Got it. I will give it another try. Thanks for the video.
